src/services/item/plugins/published/plugins/search/meilisearch.ts

```diff
@@ -328,15 +336,17 @@ export class MeiliSearchWrapper {
   }
 
   // to be executed by async job runner when desired
-  async rebuildIndex(pageSize: number = 20) {
-    // Backfill needed pdf data - Probably remove this when everything work well after deployment
-    await this.storeMissingPdfContent(buildRepositories());
+  async rebuildIndex(pageSize: number = 10) {
+    if (MEILISEARCH_STORE_LEGACY_PDF_CONTENT) {
+      // Backfill needed pdf data - Probably remove this when everything work well after deployment
+      await this.storeMissingPdfContent(buildRepositories());
+    }
 
-    this.logger.info('Starting index rebuild...');
+    this.logger.info('REBUILD INDEX: Starting index rebuild...');
     const tmpIndex = await this.getIndex(ROTATING_INDEX);
 
     // Delete existing document if any
-    this.logger.info('Cleaning temporary index...');
+    this.logger.info('REBUILD INDEX: Cleaning temporary index...');
```

Comment:
Here you delete the previous temporary index right?

Reply:
Yes, But I prefix all the log with the same string so we can filter for it as a single "process"